src/test/java/test/listeners/ExecutionListenerAndSuiteListenerTest.java

```diff
@@ -0,0 +1,38 @@
+package test.listeners;
+
+import static org.testng.Assert.assertEquals;
+
+import java.io.IOException;
+import java.util.List;
+
+import javax.xml.parsers.ParserConfigurationException;
+
+import org.testng.TestNG;
+import org.testng.TestNGException;
+import org.testng.annotations.Test;
+import org.testng.xml.Parser;
+import org.testng.xml.XmlSuite;
+import org.xml.sax.SAXException;
+
+
+public class ExecutionListenerAndSuiteListenerTest {
+
+    @Test
+    public void executionListenerAndSuiteListenerTest() {
+        String suiteFile = "src/test/resources/executionlistenersingletoncheck/parent.xml";
+        runTests(suiteFile);
+    }
+
+    private static void runTests(String suiteFile) {
+        List<XmlSuite> suites;
```

Comment:
Can I ask to add a `private static List<XmlSuite> getSuites(String... suiteFiles)`?

It could be added directly in `SimpleBaseTest` too :)

Reply:
sure, which makes it neat. :+1: